tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_access.c

```diff
@@ -327,6 +452,41 @@ static void test_sub_ext_model(void)
 	PASS();
 }
 
+static void test_sub_capacity_ext_model(void)
+{
+	bt_mesh_test_cfg_set(NULL, WAIT_TIME);
+	bt_mesh_device_setup(&prov, &local_comp);
+	provision(UNICAST_ADDR2);
+	common_configure(UNICAST_ADDR2);
+
+	uint8_t status;
+	int err;
+
+	for (int i = 0; i < CONFIG_BT_MESH_MODEL_GROUP_COUNT; i++) {
```

Comment:
Could you add comments for these 2 for-loops what is expected behavior?
I would also add a check that the next attempt to subscribe model 2 to some address fails (same for model 3 and model 1).

Reply:
redesigned this test